Tests/TestEvents.swift

```diff
@@ -239,7 +239,7 @@ class TestEvents: XCTestCase {
         event.subtitle = "subtitle"
         event.sound = "sound"
         event.launchImageName = "image"
-        event.userInfo = userInfo as [String : NSObject]
+        event.userInfo = userInfo
         event.attachments = attachments as [NSObject]
```

Comment:
NSObject here and below

Reply:
The `attachments` we will also need to keep as `NSObject` because we assign them from an OS representation which inherits from `NSObject`.